packages/tables/resources/views/components/summary/row.blade.php

```diff
@@ -34,7 +39,7 @@
 
     @if ($extraHeadingColumn || $groupsOnly)
         <x-filament-tables::cell
-            class="text-sm font-medium text-gray-950 dark:text-white"
+            class="px-3 text-sm font-medium text-gray-950 dark:text-white"
```

Comment:
Can you send some screenshots of the design change here please?

Reply:
Agree we need to be careful here.
If additional spacing is needed, it should probably be added to a wrapper around the heading instead, as `px-3` collides with the horizontal spacing classes on the cell component that depend on screen size.

Reply:
To me it seems like the `Type` heading has too much padding actually?

Reply:
no actually I believe is that the empty `td` which are meant for checkboxes is still in the table 
![image](https://github.com/filamentphp/filament/assets/67364036/dbd21dac-f201-4a77-ae05-8c68ece15423)

of course the ideal one probably is to remove that checkbox column, but I think that requires too much of changes for this PR that I'm comfortable to make